bgbb_airflow/pred_airflow_job.py

```diff
@@ -97,6 +104,14 @@ def transform(df, bgbb_params, return_preds=(14,)):
     for days, udf in n_returns_udfs:
         df2 = df2.withColumn(days, udf(df.Frequency, df.Recency, df.N))
     df2 = add_p_th(bgbb, dfs=df2, fcol="Frequency", rcol="Recency", ncol="N")
+    df2 = add_mau(
+        bgbb,
+        dfs=df2,
+        fcol="Frequency",
+        rcol="Recency",
+        ncol="N",
+        n_days_future=28,
+    )
```

Comment:
Could you add a test that asserts the expected value of `transform(...).columns`? I think it would make it clear what the new fields are.

Reply:
just added in 1315d32